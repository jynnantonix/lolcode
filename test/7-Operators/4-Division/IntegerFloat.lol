This test makes sure the division operator works correctly on integer and floating point decimal arguments.

HAI
	VISIBLE QUOSHUNT OF 1 AN 2.34 MKAY?
KTHXBAI
