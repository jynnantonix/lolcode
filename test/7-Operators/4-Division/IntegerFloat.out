0.4273504273504274 
